content/data_science/3_introduction_to_statistics/introduction_to_statistics/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+_db_id: 106
+title: introduction to statistics
+---
+
+# Introduction to Statistics
```

Comment:
Please always run the hugo server and make sure things look good. This has 2 top level titles that say the same thing, but that have different capitalization.